spec/devise/jwt/revocation_strategies/blacklist_spec.rb

```diff
@@ -30,4 +30,19 @@
       expect(strategy.find_by(jti: '123')).not_to be_nil
     end
   end
+
+  describe '#revoke_jwt(payload, user)' do
```

Comment:
We should add the `it` within the already existent [`describe '#revoke_jwt(payload, user)`](https://github.com/waiting-for-dev/devise-jwt/blob/master/spec/devise/jwt/revocation_strategies/blacklist_spec.rb#L26).